### ISSUE TITLE
Popup fails to open https://login.microsoftonline.com/

### ISSUE THREAD

**Issue by diamant-isufi-sveasolar** [AzureAD/microsoft-authentication-library-for-js#5296]
### Core Library

MSAL.js v2 (@azure/msal-browser)

### Core Library Version

2.28.2

### Wrapper Library

MSAL React (@azure/msal-react)

### Wrapper Library Version

1.4.6

### Public or Confidential Client?

Public

### Description

There is this issue that sometimes the popup fails to open https://login.microsoftonline.com/, instead it opens the URL that my project is hosted in(e.g: staging.test123.com), so the URL is the same in the browser and in the popup. This leads to me not being able to complete the sign in, no errors appear in the info logger.

Am I missing something in the config, or should I provide any fallback route for https://login.microsoftonline.com/.

### MSAL Configuration

```javascript
auth: {
    clientId: process.env.REACT_APP_AZURE_CLIENT_ID,
    redirectUri: process.env.REACT_APP_AZURE_REDIRECT_URI,
    postLogoutRedirectUri: process.env.REACT_APP_AZURE_POSTLOGOUT_REDIRECT_URI,
  
  },
  cache: {
    cacheLocation: "localStorage",
    storeAuthStateInCookie: isIE || isEdge || isFirefox,
  },
  system: {
    loggerOptions: {
      loggerCallback: (level, message, containsPii) => {
        if (containsPii) {
          return;
        }
        switch (level) {
          case LogLevel.Error:
            console.error(message);
            return;
          case LogLevel.Info:
            console.info(message);
            return;
          case LogLevel.Verbose:
            console.debug(message);
            return;
          case LogLevel.Warning:
            console.warn(message);
            return;
          default:
            return;
        }
      },
    },
  }
```

### Relevant Code Snippets

```javascript
<MsalAuthenticationTemplate
      interactionType={InteractionType.Popup}
      authenticationRequest={authRequest}
      errorComponent={ErrorComponent}
      loadingComponent={LoadingComponent}>
      <LoginRedirect />
</MsalAuthenticationTemplate>
```

### Identity Provider

Azure AD / MSA

### Source

External (Customer)

**Comment by tnorling** [comment]
@diamant-isufi-sveasolar Are you sure it isn't hitting login.microsoftonline.com and then immediately redirecting back to your app? In cases where you already have an active session with AAD you may be redirected back without interacting with the page. Try collecting some network logs to confirm.

**Comment by diamant-isufi-sveasolar** [comment]
@tnorling it does go to https://login.microsoftonline.com, but then it redirects me back to my app **inside** the popup, so it doesn't close.

**Comment by tnorling** [comment]
Does the url of the popup include the #code= or has the hash been stripped off?

**Comment by diamant-isufi-sveasolar** [comment]
@tnorling The login.microsoftonline route opens up, it then redirects me back to the web route, but fails to close.. I am then stuck in the popup having the route that should be in the browser. This is a screenshot of the logs

<img width="566" alt="image" src="https://user-images.githubusercontent.com/105778530/196633645-3ac7d4fe-3d60-44d1-80aa-81722b8e4b18.png">

**Comment by bmahall** [comment]
@diamant-isufi-sveasolar What are you using as authority in the authConfig? Is it a multi-tenant application
?

**Comment by hectormmg** [comment]
@diamant-isufi-sveasolar this should be fixed in version 2.30.0, please upgrade and let us know.
